email_automation/email-config.py

```diff
@@ -4,40 +4,57 @@
 import os
 import logging
 import sys
+import argparse
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 # If you don't have boto3 installed, make sure to `pip install boto3` before running this script. 
 
-#Input the filename as an argument in command line 
 if __name__ == "__main__":
-    filename_raw = sys.argv[1]
-    filename = filename_raw.split(".")[0]
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-l', '--local',
+                       help = 'Running locally. Provide full path to config file + install boto3 prior to running.' )
+    group.add_argument('-g', '--github',
+                       help = 'Must be run on GitHub. To run locally, use -l argument.') 
+    args = parser.parse_args()
+    filepath_raw = sys.argv[2]
+    filename_raw = filepath_raw.split("/")[-1]
+    filename = filename_raw.split('.')[0]
     pool_name = "nrelopenpath-prod-" + filename
-    # pool_name = 'nrelopenpath-stage'
     current_path = os.path.dirname(__file__)
-    config_path = os.path.relpath('../configs/'+ filename_raw, current_path)
+    maindir = current_path.rsplit("/",1)[0]
+    config_path = os.path.relpath('../configs/'+ filename_raw, current_path) if args.local else maindir + f'/configs/{filename_raw}'
```

Comment:
@nataliejschultz if `args.local`, then the user can specify the full path to the file. I don't see how prepending `.../configs` will work here.

I think this should be:

```suggestion
    config_path = filepath_raw if args.local else maindir + f'/configs/{filename_raw}'
```

Reply:
Committed !